applications/luci-app-ddns/htdocs/luci-static/resources/view/ddns/overview.js

```diff
@@ -674,6 +680,11 @@ return view.extend({
 					};
 				}
 
+				if (Boolean(s.url)) {
+					o = s.taboption('basic', form.DummyValue, '_url', _("Update URL"));
+					o.default = s.url;
```

Comment:
I find that monospace makes the URL easier to parse:
```js
					o.rawhtml = true;
					o.default = '<div style="font-family: monospace;">' + s.url; + '</div>'
```

Reply:
We'd have to properly escape that `s.url` as well.

If we want to show this in monospace, I'd much rather add support for `.monospace` to `form.Value` and `form.DummyValue` and configure it more cleanly that way, rather than hacking it with `o.rawhtml`. (A much bigger change, granted.)

Reply:
I had a thought that perhaps adding a whole `.style` element might be the way to go. But that's for another day. 

The whole point of `.rawhtml` is so you don't have to escape. That you provide raw HTML.

https://openwrt.github.io/luci/jsapi/LuCI.form.DummyValue.html

Reply:
Right. And if we are to provide raw HTML, it has to be a valid, properly escaped HTML, doesn't it? And we cannot know that for the URL value.

So I added some `.replace`s; rebased on master.

Reply:
Those only have to be escaped in attributes.